test/app-tap/tarantoolctl.test.lua

```diff
@@ -111,10 +112,11 @@ local function tctl_wait_stop(dir, name)
 end
 
 local function tctl_command(dir, cmd, args)
+    local rocks_path = SOURCEDIR .. '/third_party/luarocks/src/?.lua;'
     if not fio.stat(fio.pathjoin(dir, '.tarantoolctl')) then
         create_script(dir, '.tarantoolctl', tctlcfg_code)
     end
-    local command = ('tarantoolctl %s %s'):format(cmd, args)
+    local command = ('LUA_PATH=\"' .. rocks_path .. '\" tarantoolctl %s %s'):format(cmd, args)
```

Comment:
Nit: Backslashes are not necessary here.

Reply:
fixed